scripts/build.sh

```diff
@@ -25,7 +25,7 @@ function build_archive {
   rm -f $TLDR_ARCHIVE
 
   cd $TLDRHOME/
-  zip -r $TLDR_ARCHIVE pages/ LICENSE.md
+  zip -r $TLDR_ARCHIVE pages/ pages.*/ LICENSE.md
```

Comment:
Why not just `pages*/` ?

Reply:
Does it even work with the `.` there, considering that the English pages directory does not have a suffix?

Reply:
@waldyrious, I assume it would in the initial one as it has `pages/ pages.*/` so that would catch both.